gensim/models/keyedvectors.py

```diff
@@ -1081,15 +1223,17 @@ def evaluate_word_analogies(self, analogies, restrict_vocab=300000, case_insensi
         dummy4unknown : bool, optional
             If True - produce zero accuracies for 4-tuples with out-of-vocabulary words.
             Otherwise, these tuples are skipped entirely and not used in the evaluation.
+        topk: int, optional
+            Number of the top similar words we can consider for a successful prediction.
+        method: str, by default equal to 3CosAdd.
```

Comment:
```suggestion
        method: str, optional
```

You should also mention the default value (3CosAdd) as well as any other acceptable values below.